code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -301,7 +301,9 @@
 		return FALSE
 	// Time to set up our team structure
 	if(length(thunderdome_candidates) > max_thunderdome_players)
-		thunderdome_candidates.Cut(max_thunderdome_players)
+		thunderdome_candidates.Cut(max_thunderdome_players + 1)
+	if(ISODD(length(thunderdome_candidates))) // We want fair fights
+		pick_n_take(thunderdome_candidates)
```

Comment:
can you make this send a message to the removed candidate and let them know that they were cut from the list automatically due to there being an odd number of participants?

Reply:
Done